src/datasources/data-frame/query_help.md

```diff
@@ -0,0 +1,74 @@
+# SystemLink Data Frames data source
+
+## 📊 Overview
+
+The **SystemLink Data Frames** data source allows you to display data table properties and visualize data from multiple data tables in dashboards.
+
+---
+
+## 🎯 Key Features
+
+* **Flexible Filtering**: Filter data tables by result, data table, and column properties.  
+* **Display Table Metadata**: Retrieve and display metadata properties such as name, ID, row count, column information, and workspace.  
+* **Display Data Table Data**: Retrieve and display row data from selected columns across one or more filtered data tables.  
+
+---
+
+## ⚠️ Limitations
+
+### Required Filters
+At least one filter (data table, test result, or column filter) must be specified. Queries without any filters will return no results.
```

Comment:
Suggestion: 'You must specify at least one filter such as a filter for data tables, test results, or columns. A query without a filter does not return results.'